docs/UserGuide.md

```diff
@@ -83,19 +83,22 @@ You can find out more about the command terminologies here:
 
 ### Adding an assignment: `add`
 
-Adds an assignment into your schedule.
+Format: `add n/NAME_OF_ASSIGNMENT d/DEADLINE_OF_ASSIGNMENT TIME_ASSIGNMENT_IS_DUE mod/MODULE​[priority/PRIORITY] 
+[remind]`
 
-Format: `add n/NAME_OF_ASSIGNMENT d/DEADLINE_OF_ASSIGNMENT TIME_ASSIGNMENT_IS_DUE mod/MODULE​ [remind]`
+You can add your assignments into your schedule so that you can manage all your assignments and
+ academic tasks conveniently. 
 
```

Comment:
It may be good to mention where the user can find their added assignments :)

Reply:
Are you referring to the different components like the commandbox, remindersbox etc ? What are their names ah cause the labelled diagram is not up yet

Reply:
Yup. Umm not sure tbh.

You can either ask @andreatanky or next to "assignment list" just put "(highlighted in red)" or whatever colour you want to highlight it :)